lib/proxy/log_buffer/decorator.rb

```diff
@@ -50,7 +50,7 @@ def add(severity, message = nil, progname = nil, backtrace = nil)
       end
       info("Logging file reopened via USR1 signal") if reopened
       # exceptions are also sent to structured log if available
-      self.exception("Error details", backtrace) if backtrace && backtrace.is_a?(Exception)
+      self.exception("Error details", backtrace) if backtrace&.is_a?(Exception)
```

Comment:
Is this actually needed? Doesn't `is_a?` always work, even on nil?

Reply:
Correct.